README.md

```diff
@@ -91,9 +91,58 @@ suite.reporter = DotReporter
 suite.run()
 ```
 
+#### Using the options map on the static `test` method:
+
+```js
+import "./testie" for Testie
+import "./src/tap_reporter" for TapReporter
```

Comment:
I think perhaps we need:

```
import "wren-testie/reporters" for TapReporter
```

Another choice would be to import them all into Testie but since there are limits on the total # of classes I think perhaps having a reporters namespace might be reasonable?  

This `./src/tap_reporter` doesn't work for projects pulling this in as a library dependency... everything needs to be relative to `lwren-testie`.